Feature #3122 v6.2.0-rc1

georgemccabe opened the pull request:
Review release notes: https://metplus.readthedocs.io/en/feature_3122_6.2.0-rc1/Users_Guide/release-notes.html

georgemccabe committed:
add more release notes

georgemccabe committed:
Merge branch 'develop' into feature_3122_6.2.0-rc1

georgemccabe committed:
update the rc1 date

georgemccabe committed:
update the DockerHub 6.2.0-rc1 images and dates

michelleharrold reviewed (dismissed):
@georgemccabe -- this looks great. I have two quick questions/comments: 1) Would it be possible under the New Use Cases dropdown to change Credit --> CREDIT (I know it is linked to the issue, and this is minor. So no worries if this is a pain.), and 2) We recently deactivated our METplus AWS account, so I am wondering if for the official release, we remove the AWS install area?

georgemccabe committed:
change Credit to CREDIT

georgemccabe committed:
remove AWS existing build info because we no longer provide those images

georgemccabe commented:
@michelleharrold, both good points. I removed the AWS references from the Existing Builds page and template. I had renamed the GitHub issue to changed Credit to CREDIT, but I forgot to update the release notes. Thanks for catching that!
